examples/approval_request/main.go

```diff
@@ -14,15 +14,15 @@ import (
 )
 
 const (
-	myAPIKey string = "M2VkNjg4ODgtMTFmOC00YTNiLTg0NmEt..."
-	keyName  string = "very important key"
+	myAPIKey string = "NjNmNmRlM2ItYjQ4YS00YjJkLWJkNGQtY2Y2Y2Y4YTEwOGVlOm9mYzdKdzhpVmJEaGZBTzU2aWtCNzNGV0pJYlhkMkE4RnJvLXo5eE53LUI3Zkp5WWlSNWIxdWZ5a0E3ekNkMWlDSGYxcjg2aUNDR29qNmE2aEpFcGJB"
```

Comment:
If this is just an example, please replace this with some meaningful words to be replaced.
```suggestion
	myAPIKey string = "Please put your API key here"
```

Reply:
If you are using a real key, you already leaked it, please remove it.